recipes/freediscovery/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "Freediscovery" %}
+{% set version = "0.9.1" %}
+{% set sha256 = "15bc9b098f30e79ac39b803566f2e84c392abf5f0acb54dad434952997fef7b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{version}}.tar.gz
```

Comment:
You might consider releasing this on pypi so you can pull it from there instead of from github. Oddly, the shas from github are not guaranteed to be stable over time. We've seen this issue in a number of feedstocks. [Here's one example](https://github.com/conda-forge/pandas-feedstock/pull/19#issuecomment-276019321). It's definitely ok to pull the archive from github though, so if you'd prefer to keep this as is, then this PR LGTM.